<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelNpcFilePath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>Npc.xlsx 路径</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlotFilePath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>Plot.xlsx 路径</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSceneFilePath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>Scene.xlsx 路径</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFindNpc">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>28</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFindPlot">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>28</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFindScene">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>28</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textNpcPath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPlotPath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textScenePath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
